notebooks/official/custom/SDK_Custom_Container_Prediction.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>You can use Cloud Build instead of Docker to build, so switch to that. That will work in Colab.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/503/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>https://cloud.google.com/build/docs/build-push-docker-image</p>

Reply:
updated ! replaced docker-build with gcloud-builds and excluded the rest of the optional docker steps from COLAB & testing environment.